src/test/java/org/folio/rest/api/CirculationSettingsAPITest.java

```diff
@@ -1,27 +1,50 @@
 package org.folio.rest.api;
 
-import static org.hamcrest.CoreMatchers.is;
-import static org.hamcrest.junit.MatcherAssert.assertThat;
+import io.vertx.core.json.JsonObject;
+import org.folio.rest.support.ApiTests;
+import org.folio.rest.support.http.AssertingRecordClient;
+import org.folio.rest.support.http.InterfaceUrls;
+import org.junit.Before;
+import org.junit.Test;
 
 import java.net.MalformedURLException;
 import java.util.UUID;
 import java.util.concurrent.ExecutionException;
 import java.util.concurrent.TimeoutException;
 
-import org.folio.rest.support.ApiTests;
-import org.folio.rest.support.http.AssertingRecordClient;
-import org.folio.rest.support.http.InterfaceUrls;
-import org.junit.Test;
-
-import io.vertx.core.json.JsonObject;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.junit.MatcherAssert.assertThat;
 
 public class CirculationSettingsAPITest extends ApiTests {
 
+  private static final String TABLE_NAME = "circulation_settings";
   private final AssertingRecordClient circulationSettingsClient =
     new AssertingRecordClient(
-    client, StorageTestSuite.TENANT_ID, InterfaceUrls::circulationSettingsUrl,
+      client, StorageTestSuite.TENANT_ID, InterfaceUrls::circulationSettingsUrl,
       "circulation-settings");
 
+  @Before
+  public void beforeEach() {
+    StorageTestSuite.cleanUpTable(TABLE_NAME);
+  }
+
+  @Test
+  public void updateInsteadCreateWithTheSameName() throws MalformedURLException,
+    ExecutionException, InterruptedException, TimeoutException {
+    String id = UUID.randomUUID().toString();
+    JsonObject circulationSettingsJson = getCirculationSetting(id);
+    circulationSettingsClient.create(circulationSettingsJson);
```

Comment:
I think it makes sense after the first creation to check that there is one entity in the storage and to verify the id and name

Reply:
done